tests/angular/package.json

```diff
@@ -1,20 +1,45 @@
 {
   "name": "@ni/angular-test",
   "version": "1.0.0",
-  "private": true,
   "description": "Smoke test for Angular project integration.",
-  "type": "module",
   "scripts": {
-    "test": "eslint ."
-  },
-  "peerDependencies": {
-    "@ni/eslint-config-angular": "*",
-    "@ni/eslint-config-javascript": "*"
+    "start": "ng serve",
+    "build": "ng build && ng build angular-library",
+    "lint": "ng lint",
+    "ng-test": "ng test --watch=false",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng lint && eslint . && cd projects/angular-library && eslint ."
   },
+  "private": true,
   "dependencies": {
-    "@angular/core": "^18.2.13"
+    "@angular/animations": "^18.2.0",
+    "@angular/common": "^18.2.0",
+    "@angular/compiler": "^18.2.0",
+    "@angular/core": "^18.2.0",
+    "@angular/forms": "^18.2.0",
+    "@angular/platform-browser": "^18.2.0",
+    "@angular/platform-browser-dynamic": "^18.2.0",
+    "rxjs": "~7.8.0",
+    "tslib": "^2.3.0",
+    "zone.js": "~0.14.10"
   },
   "devDependencies": {
-    "@types/jasmine": "^5.1.4"
+    "@angular-devkit/build-angular": "^18.2.21",
+    "@angular/cli": "^18.2.21",
+    "@angular/compiler-cli": "^18.2.0",
+    "@angular/localize": "^18.2.14",
```

Comment:
We are imminently working on Angular 19 upgrade, thoughts on just jumping straight to that?